raster/r.geomorphon/main.c

```diff
@@ -117,11 +112,11 @@ int main(int argc, char **argv)
 	G_add_keyword(_("machine vision geomorphometry"));
 
 	opt_input = G_define_standard_option(G_OPT_R_ELEV);
-	opt_input->key = rasters[0].name;
+	opt_input->key = "elevation";
 	opt_input->required = YES;
-	opt_input->description = _(rasters[0].description);
+	opt_input->description = _("Name of input elevation raster map");
```

Comment:
You can actually drop key and description here. See https://grass.osgeo.org/grass78/manuals/parser_standard_options.html

Reply:
You are right. And `required` is excess as well.

Reply:
Right, esp. because elevation is the only input, so there is no need to be explicit about elevation being required.